udata_front/theme/gouvfr/assets/js/components/vanilla/update-url.js

```diff
@@ -47,6 +47,7 @@ function moveToHash() {
   } else {
     tryPreviousHashes(hash.slice(1));
   }
+  globalThis._paq?.push(['trackEvent', 'navigation', 'Move page to hash', window.location.pathname, hash]);
```

Comment:
What do we want to track with this?

Reply:
This allows us to see tab changes on dataset and organization pages

Reply:
Somehow, it didn't seem to send any event on tab changes (Fichier, Réutilisation, etc.)